lib/react_client.dart

```diff
@@ -245,40 +318,10 @@ ReactComponentFactory _registerComponent(ComponentFactory componentFactory, [Ite
     )])
   ]);
```

Comment:
I like the idea of this, but I tried it out and there was a pretty big performance drop (30% slower)